core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -66,9 +68,17 @@ class AdminClient(val time: Time,
     throw new RuntimeException(s"Request $api failed on brokers $bootstrapBrokers")
   }
 
-  def findCoordinator(groupId: String): Node = {
+  var timerMs: Long = 0
```

Comment:
I think there is a way to remove the global variable with the current method signatures. I'll submit that in the next update. If it turns out that's not the best approach I could switch to using your suggestion.